src/pool/srv_pool.c

```diff
@@ -681,6 +695,25 @@ pool_prop_write(struct rdb_tx *tx, const rdb_path_t *kvs, daos_prop_t *prop)
 	return rc;
 }
 
+static uint32_t
+get_svc_ops_age(void)
+{
+	const uint32_t ops_age_default = DEFAULT_PS_OPS_ENTRY_AGE_SEC;
+	char          *v;
+	int            n;
+
+	v = getenv(DAOS_MD_OPS_AGE_SEC_ENV); /* in MB */
```

Comment:
do we want to be exposing more environment variables rather than adding to the server config parameters? is this documented?

Reply:
@mjmac @NiuYawei should this be changed to a server config parameter?

Reply:
I had a chance to discuss this with @mjmac and @liw today. I will in a subsequent patch take a look at enhancing this (eliminating this variable), and also the existing rdb size (DAOS_MD_CAP) environment variable. Rather than using an environment variable or a daos_server.yml parameter interface - that would apply the same value to all pools created, to allow customization per pool via an immutable pool property. So these values could be specified as properties input to pool create. And the properties could be queried (but not set) subsequently.